lang/java/avro/src/main/java/org/apache/avro/data/Json.java

```diff
@@ -55,8 +56,12 @@ private Json() {}                               // singleton: no public ctor
   public static final Schema SCHEMA;
   static {
     try {
```

Comment:
Currently, Avro supports 1.6 so we are not able to use try-with-resources, unfortunately.